ee/desktop/notify/notify_darwin.m

```diff
@@ -0,0 +1,46 @@
+//go:build darwin
+// +build darwin
+
+// Draws from Fyne's implementation: https://github.com/fyne-io/fyne/blob/master/app/app_darwin.m
+
+#import <Foundation/Foundation.h>
+#import <UserNotifications/UserNotifications.h>
+
+void doSendNotification(UNUserNotificationCenter *center, NSString *title, NSString *body) {
+    UNMutableNotificationContent *content = [UNMutableNotificationContent new];
+    [content autorelease];
+    content.title = title;
+    content.body = body;
+
+    NSString *uuid = [[NSUUID UUID] UUIDString];
+    NSString *identifier = [NSString stringWithFormat:@"kolide-notify-%@", uuid];
+    UNNotificationRequest *request = [UNNotificationRequest requestWithIdentifier:identifier
+        content:content trigger:nil];
+
+    [center addNotificationRequest:request withCompletionHandler:^(NSError * _Nullable error) {
+        if (error != nil) {
+            NSLog(@"Could not send notification: %@", error);
```

Comment:
Will this log go anywhere useful?

Reply:
Woops, updating that branch made the links no to to the right parts of code, updated